### PR TITLE
Add a CODEOWNERS file

### PR DESCRIPTION
**Pull request by shayfrendt**
Can someone help me route this to the right maintainer?

I am trying to figure out in what capacity this repository is used, whether it's used in production, and whether we need to mark it in or out of scope for SOC 2.

Tests__All_-_Vanta

### CODE REVIEW

**Comment by schani** [comment]
@shayfrendt  This is very out of scope for SOC2.  It's a CLI tool for helping with maintaining Glide's packages.  It's not in production, or even in CI.  Developers run it locally on their machines to modify code.  Whatever code changes it produces will become regular PRs.

**Review by schani — APPROVED** [review]
I don't know what this empty file does, but if it makes some tool happy...

**Comment by shayfrendt** [comment]
Ok cool thanks for the additional context @schani. I will mark this repo out of scope in Vanta!

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This is a CODEOWNERS file
+# It specifies who is maintaining the code in this repository
+
+* @cif @schani
```
Comment (shayfrendt) [inline CODEOWNERS:4]: @schani Sorry this is what I meant to propose. What would you change?

Comment (schani) [reply CODEOWNERS:4]: Works for me!

**Pull request merged**